tempesta_fw/http_parser.c

```diff
@@ -3278,6 +3449,17 @@ tfw_http_parse_req(void *req_data, unsigned char *data, size_t len)
 				__FSM_MOVE_n(RGen_OWS, 7);
 			}
 			__FSM_MOVE(Req_HdrP);
+		case 'r':
+			if (likely(__data_available(p, 8)
+				   && C4_INT_LCM(p + 1, 'e', 'f', 'e', 'r')
+				   && TFW_LC(*(p + 5)) == 'e'
+				   && TFW_LC(*(p + 6)) == 'r'
+				   && *(p + 1) == ':'))
```

Comment:
Perhaps, this line should look like this? Is this code tested at all?
```C
&& *(p + 7) == ':'))
```

Reply:
Corrected.